post-install-configure.sh

```diff
@@ -1,37 +1,29 @@
 #!/bin/bash
  
-sed -e "s/10.252.0.7/$(hostname -i)/" >/etc/systemd/system/cray-node-exporter.service <<'EOF'
+cat  > /etc/systemd/system/cray-node-exporter.service <<EOF
 
 [Unit]
 Description=Node Exporter
 
 [Service]
 Type=simple
 ExecStart=/usr/bin/node_exporter \
---web.listen-address=10.252.0.7:9100 \
---collector.buddyinfo \
---collector.drbd \
---collector.interrupts \
---collector.ksmd \
---collector.logind \
---collector.meminfo_numa \
---collector.mountstats \
---collector.processes \
---collector.qdisc \
---collector.systemd \
---collector.tcpstat
+--web.listen-address=10.252.1.23:9100 \
```

Comment:
question - should this IP be the IP of the host it's being installed on?

Reply:
yes correct!